src/run/uploader/upload_metadata.rs

```diff
@@ -60,9 +61,9 @@ mod tests {
         };
 
         let hash = upload_metadata.get_hash();
-        assert_eq!(
+        assert_snapshot!(
             hash,
-            "161a1a3eeea6d988909142e1e7bae3339b3698aaeb025641aa63809895336ae7"
+            @"b672792aac09f4331b387bf66a5e9225b6649e1e3bedacc6fa20e83ceb9956d6"
         );
```

Comment:
I think this test has a counterpart in our backend to make sure our hashing is consistent on both side.
And that might be the reason behind this not being a snapshot already. If that's the case, we should keep this an assert_eq and add a comment mentioning the relation with our backend

Reply:
I'll mention this here, and change the backend value in the related backend PR